### ISSUE TITLE
Missing driver and constructor championship standings

### ISSUE THREAD

**Issue by tehaandev** [br-g/openf1#115]

**Comment by vitoramaral10** [comment]
I'd like to have this information too, I'd like to make a win probability calculator

**Comment by shayanekat** [comment]
https://openf1.org/#roadmap it's in the roadmap. I am interested too

**Comment by brzzdev** [comment]
@br-g Fantastic API.

I see this is top of the roadmap currently, and probably the most pressing omission?

Is there any timeline or update on when this might be added?

**Comment by br-g** [comment]
> [@br-g](https://github.com/br-g) Fantastic API.
> 
> I see this is top of the roadmap currently, and probably the most pressing omission?
> 
> Is there any timeline or update on when this might be added?

Thanks for the kind words — really appreciate it!
You’re absolutely right, this is one of the key missing pieces and it’s high on the list. The challenge is that it involves integrating data from an additional source, which adds some complexity. That said, it’s definitely on my radar, and I’m actively exploring options. No firm timeline just yet, but it’s something I’m seriously considering!

**Comment by JeffreyJPZ** [comment]
@br-g Apologies if this is currently being worked on, but after adding the [F1 scraping service](https://github.com/br-g/openf1/tree/main/src/openf1/services/f1_scrapping) I'd assume extending it to include driver/constructor standings isn't too far-fetched? Alternatively, if live standings are a concern the [```ChampionshipPrediction```](https://livetiming.formula1.com/static/2025/2025-08-03_Hungarian_Grand_Prix/2025-08-03_Race/ChampionshipPrediction.json) topic from the F1 Live Timing API seems useful, although you'd need to get all drivers for a team for that season to calculate constructor standings (thanks to Red Bull's 2nd seat).

**Comment by br-g** [comment]
Yes you're right @JeffreyJPZ. I haven't started yet, but I'm planning to do it soon, using scrapping.
Let me know if you want to take this task, that would really help a lot!
